### PR TITLE
requirements: Move nose and sphinx to dev requirements. See #230

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,2 +1,6 @@
+nose>=1.3.7
+nose-parameterized>=0.5.0
 flake8
 flake8-import-order
+sphinx>=1.4
+sphinx_rtd_theme>=0.2.2
```
Comment (JoostJM) [inline requirements-dev.txt]: would this require user to install the dev version of pyradiomics before documentation can be built? (or have users install `sphinx` and `sphinx-rtd-theme` manually?)

**Comment by jcfr** [comment]
> install the dev version of pyradiomics

What do you mean by dev version ? 

To summarize, as it was before building the doc can be achieved doing:

```
git clone https://github.com/Radiomics/pyradiomic
cd pyradiomic
pip install -r requirements.txt  -r requirements-dev.txt
cd docs; make html
```

The advantage of this commit is that it avoids installing sphinx related packages while simply doing `pip install pyradiomics`.

I would also suggest to remove this line: https://github.com/Radiomics/pyradiomics/blob/d00212cd407d96604d5d9f1581661547efd3be3a/MANIFEST.in#L15

**Comment by JoostJM** [comment]
> To summarize, as it was before building the doc can be achieved doing:

Building the docs does require pyradiomics to be installed, as it uses the __version__ of pyradiomics to specify the package version when building the docs.

This is not needed for the online documentation on readthedocs, but it is reflected in the downloadable pdf or custom build documentation (this was set at 0.0.1). I changed this in #216 to use the actual version of pyradiomics.

**Comment by JoostJM** [comment]
> What do you mean by dev version ?

`python setup.py develop`? Or when are the development requirements installed by the setup script?

**Comment by JoostJM** [comment]
> I would also suggest to remove this line: https://github.com/Radiomics/pyradiomics/blob/d00212cd407d96604d5d9f1581661547efd3be3a/MANIFEST.in#L15

I agree

**Comment by JoostJM** [comment]
@jcfr, can you update the manifest and merge?

**Comment by JoostJM** [comment]
@jcfr, what is the status on this PR?

**Comment by JoostJM** [comment]
Rebased onto master to resolve conflict (due to the removal of the `tqdm` dependency in #232)

**Pull request merged**